src/pages/home/report/ReportActionCompose.js

```diff
@@ -446,6 +441,48 @@ class ReportActionCompose extends React.Component {
         ];
     }
 
+    /**
+     * Build the suggestions for mentions
+     * @param {Object} personalDetails
+     * @param {String} [searchValue]
+     * @returns {Object}
+     */
+    getMentionOptions(personalDetails, searchValue = '') {
+        const suggestions = [];
+
+        if (CONST.MENTION_SUGGESTER.HERE_TEXT.includes(searchValue)) {
+            suggestions.push({
+                text: CONST.MENTION_SUGGESTER.HERE_TEXT,
+                alternateText: this.props.translate('mentionSuggestions.hereAlternateText'),
+                icons: [
+                    {
+                        source: Expensicons.Megaphone,
+                        type: 'avatar',
+                    },
+                ],
+            });
+        }
+        const sortedPersonalDetails = _.sortBy(_.values(personalDetails), (detail) => detail.displayName || detail.login);
+        _.each(sortedPersonalDetails, (detail) => {
+            if (searchValue && !`${detail.displayName} ${detail.login}`.trim().includes(searchValue)) {
```

Comment:
There's also a bug with search being case-sensitive. I think this can be resolved by calling `.toLocaleLowerCase()` for both `searchValue` and `${detail.displayName} ${detail.login}`

Reply:
I _think_ what he is saying is to only sort the matches, instead of sorting the whole personalDetails object. Since the results will always be a subset.

Reply:
> I think adding it will cause the search results to be incorrect.

I think I'm missing something, could you explain how? Essentially what I was trying to achieve is to stop iterating over `personalDetails` once we already have 5 items in suggestions array